tests/single_controller/test_high_level_scheduling_api.py

```diff
@@ -17,7 +17,7 @@
 import ray
 
 from verl.single_controller.base.worker import Worker
-from verl.single_controller.ray.base import RayClassWithInitArgs, RayResourcePool, RayWorkerGroup, merge_resource_pool
+from verl.single_controller.ray.base import RayClassWithInitArgs, RayResourcePool, RayWorkerGroup, split_resource_pool
```

Comment:
Does this mean `merge_resource_pool` is no longer supported?

Reply:
Yes, since resource pool contains only one placement group, we can't merge two resource pools. I have checked that both open source and internal repos depending verl not using `merge_resource_pool`.